deploy/modules/aws/bastion/variables.tf

```diff
@@ -0,0 +1,38 @@
+variable "bastion_name" {
+  description = "Name of the EKS cluster. Also used as a prefix in names of related resources."
+  type = string
+}
+
+variable "enable_ssh_to_workers" {
+  description = "Whether enable ssh from bastion to workers, if true, the worker_security_group_id must be provided"
+  default = false
```

Comment:
Can we enable this by default?

Reply:
No, the `worker_security_group_id` must be provided with this enabled, which can not have a default value. So user will be forced to provide the id or disable this explicitly if we enable this by default.